README.md

```diff
@@ -1,5 +1,3 @@
-#
```

Comment:
wasn't this added by @wolfenrain for linting reasons?

Reply:
It was added by me, and it was wrong. :sweat_smile:

Reply:
We have a lot of linting problems in the MD files, but an empty header wasn't the solution to the problem I tried to fix with it. Instead it just showed the `#` on pub.